SonicMania/Objects/Common/Water.c

```diff
@@ -1131,10 +1131,10 @@ void Water_State_Bubbler(void)
     }
 
     if (self->visible && RSDK.CheckOnScreen(self, &self->updateRange)) {
-        if (!self->countdownID) {
+        if (--self->countdownID <= 0) {
```

Comment:
From Steam:
![image](https://github.com/user-attachments/assets/e674a574-2cbc-4751-8a24-7f8d137d30e8)
```suggestion
        if (--self->countdownID < 0) {
```